test/utility/mockApp.ts

```diff
@@ -2,8 +2,8 @@ import * as models from 'powerbi-models';
 
 export interface IApp {
   // Load
-  load(config: models.ILoadConfiguration): Promise<void>;
-  validateLoad(config: models.ILoadConfiguration): Promise<models.IError[]>;
+  load(config: models.IReportLoadConfiguration): Promise<void>;
+  validateLoad(config: models.IReportLoadConfiguration): Promise<models.IError[]>;
```

Comment:
Well at the time I thinking it would be best to reduce the number of changes and was taking advantage of fact that Dashboard confirguration is also a valid Report configuration but I think you are right. It is probably better to explicitly support both to avoid confusion.